src/DS/rds.c

```diff
@@ -601,6 +602,13 @@ static void ds_rds_add_xccdf_test_results(xmlDocPtr doc, xmlNodePtr reports,
 {
 	xmlNodePtr root_element = xmlDocGetRootElement(xccdf_result_file_doc);
 
+	if (root_element->ns && root_element->ns->href &&
+			oscap_str_endswith((const char*)root_element->ns->href, "xccdf/1.1")) {
+		dW("Exporting ARF from XCCDF 1.1 is not allowed by SCAP specification. "
+		   "The resulting ARF will not validate. Convert the input to XCCDF 1.2 "
```

Comment:
Convertion of XCCDF 1.1 to XCCDF 1.2 is done through `xccdf_1.1_to_1.2.xsl`, right? Should we point users to this?